tools/docs/generate_api_rst.py

```diff
@@ -48,6 +48,10 @@ def main():
         if canonical is None:
             continue
 
+        # Skip copying empty files.
+        if (os.path.getsize(rst_file_path) == 0):
```

Comment:
```suggestion
        if os.path.getsize(rst_file_path) == 0:
```

Reply:
get it. i will create new commit for it.